packages/gallery/src/components/gallery/index.tsx

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
 import { BlueprintsManager, GALLERY_CONSTS, utils } from 'pro-gallery-lib';
 import ProGallery from './proGallery/proGallery';
-// import { Dimensions } from '../../common/interfaces/Dimensions';
-
-import { GalleryProps, GalleryState } from './gallery';
 import shouldValidate from './typeValidator/shouldValidate';
+import { Dimensions } from '../../common/interfaces/Dimensions';
+
+export interface GalleryProps {
```

Comment:
Can you move the interfaces to a separate file? I want types declared outside of the code itself. You can make it a normal TS file if it's easier for you (doesn't have to be a `d.ts` file if it creates problems)